ethcore/src/miner/miner.rs

```diff
@@ -865,6 +865,26 @@ impl miner::MinerService for Miner {
 		self.params.read().gas_range_target.0 / 5
 	}
 
+	fn set_minimal_gas_price(&self, new_price: U256) -> Result<bool, ()> {
+		if cfg!(feature = "price-info") {
+			warn!("Can't update fixed gas price while automatic gas calibration is enabled.");
+			return Err(());
+		}
+
+		trace!(target: "miner", "minimal_gas_price: recalibrating fixed...");
+		*self.gas_pricer.lock() = GasPricer::new_fixed(new_price);
+
+		let txq = self.transaction_queue.clone();
+		let mut options = self.options.pool_verification_options.clone();
+		self.gas_pricer.lock().recalibrate(move |gas_price| {
```

Comment:
Is it worth keeping it DRY? and re-using the same method as for per-block update?

Reply:
We only call `recalibrate()` twice, so for now I'm going to say no unless somebody else has a strong opinion about it.